test/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ if (CNL_SANITIZE)
     set(SANITIZE_LINKER_FLAGS "${SANITIZE_ENABLED_LINKER_FLAGS}")
 endif (CNL_SANITIZE)
 
-set(COMMON_CXX_FLAGS "${MISC_FLAGS} ${EXCEPTION_FLAGS} ${INT128_FLAGS} ${IOSTREAMS_FLAGS}")
+set(COMMON_CXX_FLAGS "${MISC_FLAGS} ${EXCEPTION_FLAGS} ${INT128_FLAGS} ${IOSTREAMS_FLAGS} -fconstexpr-ops-limit=200000000")
```

Comment:
Hi Dmitry. Thanks for the contribution. However, I try and keep the CMakeLists.txt files clear of toolchain-specific details in order to maximise portability. Hence I put settings like these in the [toolchain](https://cmake.org/cmake/help/v3.20/manual/cmake-toolchains.7.html) files ([example](https://github.com/johnmcfarlane/cnl/blob/main/test/toolchain/gcc.cmake#L3)).

It's an unfortunate fact that the test suite won't build for most compilers without certain flags being passed in. It's not ideal. I think I should probably change this so that the offending tests are `#ifdef`ed out, with a flag that is added to the toolchain files to reenable them. (I've been meaning to do something like this anyway so I'll prioritise this as a user pain point.)

To get the tests to build, you can pass the [`CMAKE_TOOLCHAIN_FILE`](https://cmake.org/cmake/help/v3.20/variable/CMAKE_TOOLCHAIN_FILE.html) option into `cmake`:

```bash
cmake -DCMAKE_TOOLCHAIN_FILE=global/path/to/cnl/test/toolchain/gcc.cmake path/to/cnl
```

or via Conan:

```bash
conan install --env CONAN_CMAKE_TOOLCHAIN_FILE=global/path/to/cnl/test/toolchain/gcc.cmake path/to/cnl
```

(Note that the toolchain path cannot be relative to the current directory.)

Reply:
> To get the tests to build, you can pass the CMAKE_TOOLCHAIN_FILE option into cmake:

Thanks, I'll try it!